app/services/attachments/concatenation_service.rb

```diff
@@ -119,7 +119,12 @@ def validate_paired_end_files(attachments)
 
     # Validates if the file extensions all match for the attachments
     def validate_file_extensions(attachments)
```

Comment:
Instead of validating the `extension` why not just validate the `format`. I.e. if `format` and `compression` match then go ahead and concatenate all the files. We shouldn't prevent `.fq. and `.fastq` files from being concatenated.

Reply:
Updated to validate the file format based on format and compression in 0edf68ed95c38f320ab27504b13a339494c776ce